src/libs/actions/IOU.js

```diff
@@ -1392,6 +1392,14 @@ function setMoneyRequestParticipants(participants) {
     Onyx.merge(ONYXKEYS.IOU, {participants});
 }
 
+/**
+ * @param {String} receiptPath
+ * @param {String} receiptSource
+ */
+function setMoneyRequestReceipt(receiptPath, receiptSource) {
+    Onyx.merge(ONYXKEYS.IOU, {receiptPath, receiptSource});
```

Comment:
We have a reset function `resetMoneyRequestInfo`. We should reset `receiptPath` as well.